ffmpeg_normalize/__main__.py

```diff
@@ -249,6 +249,22 @@ def create_parser() -> argparse.ArgumentParser:
         ),
     )
 
+    group_ebu.add_argument(
+        "--safe-ebu",
+        type=float,
+        help=textwrap.dedent(
+            """\
+        Automatically lower EBU Integrated Loudness Target to prevent falling
```

Comment:
And the option value is a relative one, right?

So it should say "automatically lower EBU Integrated Loudness Target _by this value_" or something.

Still thinking about the use case of explicitly providing a value here? Shouldn't this rather just be a boolean option and, if enabled, lower the loudness target automatically to the right value?

Reply:
Well, the second paragraph mentions "by the given amount" and the `--help` text shows that it expects a value `--safe-ebu MARGIN` so it seems clear enough?

I thought about the boolean option, but I couldn't decide what the right value would be. Sometimes a margin value of 0 works just fine, at others it falls back to dynamic again. Should we just set it to some arbitrary small value like `0.005` or something?

Reply:
I think having a default value that's small but prevents most cases of dynamic fallback is the better option!

In your tests, did 0.005 suffice? I think even 0.1 or something would not make a difference in terms of actual perceived loudness.

Reply:
A value of 0.005 seemed to work the few times I've used it but as you've said even a 0.1 is unnoticeable so better to go with that.